test/logs/main.go

```diff
@@ -0,0 +1,91 @@
+/*
+Copyright 2017 Google Inc. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"io"
+	"os"
+
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+
+	tlogs "github.com/knative/build-pipeline/test/logs/helpers"
```

Comment:
can we call the package something more specific than `helpers`? I think `helpers` is one of those package names that is hard to reason about and makes it difficult what should go there and what shouldn't (see "bad package names" at https://blog.golang.org/package-names)

The fact that you've imported this as `tlogs` I feel is a hint that maybe we don't even need the `helpers` package? We could have everything exist directly in `test/logs`

Or we could break `helpers` up into more specific packages, e.g. `test/logs/pods`, `test/logs/pipelinerun`

Reply:
Sure makes sense. I will refactor to include all logs in one package for now.